config/intelfocl_sample.json

```diff
@@ -0,0 +1,13 @@
+{
+  "TARGET" : "intelfocl",
```

Comment:
I think this should be the specific cloud fpga, instead of intelfocl or aocl.

Reply:
Agreed with the above comment

Reply:
Thank you for the comment, I understand the point here is to differentiate between Chisel-based-soc version and OpenCL-based version.

I think we may need to discuss further about the naming, as intel opencl support both Cloud FPGA and SOC (like Cyclone V)

Reply:
SO I believe that the solution was to go with "arria10" in this context?

Reply:
I believe "Arria 10" is too restrictive here. The code should work for all devices supported by Intel OpenCL for FPGA, namely Arria 10, Stratix V/10 and Cyclone V/10. That's the reason we name it as "intelfocl".
So far we have tested it on both Arria 10 and Stratix 10 boards, and it worked for both boards.

Reply:
thank you for clarifying, see main conversation thread for resolution